validatehttp/spec.py

```diff
@@ -149,6 +149,16 @@ def matches(self, resp):
                                 'Response header mismatch: {0}'.format(header),
                                 mismatch=(value, resp_value),
                             )
+                elif key == 'content':
+                    if not resp.ok:
+                        # Response was not OK. Ignoring content check.
+                        continue
+
+                    for value in params.get('content'):
+                        if value not in resp.text:
+                            raise ValidationError(
+                                'Response content not found: {0}'.format(value),
```

Comment:
```suggestion
                                f'Response content not found: {value}',
```

Reply:
I just followed the pattern style the whole file follows.

Reply:
It's never too later to modernize our codebase.